rerun_py/rerun_sdk/rerun/notebook.py

```diff
@@ -191,6 +191,48 @@ def _add_table_id(self, record_batch: RecordBatch, table_id: str) -> RecordBatch
         # Create new record batch with updated schema
         return RecordBatch.from_arrays(record_batch.columns, schema=schema)
 
+    def set_application_blueprint(
+        self, application_id: str, blueprint: BlueprintLike, make_active: bool = True, make_default: bool = True
+    ) -> None:
+        """
+        Set the blueprint for the given application.
+
+        Parameters
+        ----------
+        application_id: str
+            The ID of the application to set the blueprint for.
+        blueprint: BlueprintLike
+            The blueprint to set for the application.
+        make_active: bool
+            Whether to make the blueprint active.
+            If `True`, the blueprint will be set as the active blueprint for the application.
+        make_default: bool
```

Comment:
no need to repeat the types, and will lead to bad docs if the types do change

```suggestion
        application_id:
            The ID of the application to set the blueprint for.
        blueprint:
            The blueprint to set for the application.
        make_active:
            Whether to make the blueprint active.
            If `True`, the blueprint will be set as the active blueprint for the application.
        make_default:
```